doc/releases/changelog-dev.md

```diff
@@ -197,6 +197,32 @@
 
 <h3>Breaking changes 💔</h3>
 
+* `qml.sample` no longer has singleton dimensions squeezed out for single shots or single wires. This cuts
+  down on the complexity of post-processing due to having to handle single shot and single wire cases
+  separately. The return shape will now *always* be `(shots, num_wires)`.
+  [(#7944)](https://github.com/PennyLaneAI/pennylane/pull/7944)
+
+  For a simple qnode:
+
+  >>> @qml.qnode(qml.device('default.qubit'))
+  ... def c():
+  ...   return qml.sample(wires=0)
+
+  Before the change, we had:
+  
+  >>> qml.set_shots(c, shots=1)()
+  0
+
+  and now we have:
+
+  >>> qml.set_shots(c, shots=1)()
+  array([[0]])
+
+  Previous behavior can be recovered by squeezing the output:
+
+  >>> qml.math.squeeze(qml.set_shots(c, shots=1)())
+  0
```

Comment:
````suggestion
  ```pycon
  >>> @qml.qnode(qml.device('default.qubit'))
  ... def c():
  ...   return qml.sample(wires=0)
  ```

  Before the change, we had:
  
  ```pycon
  >>> qml.set_shots(c, shots=1)()
  0
  ```

  and now we have:

  ```pycon
  >>> qml.set_shots(c, shots=1)()
  array([[0]])
  ```

  Previous behavior can be recovered by squeezing the output:

  ```pycon
  >>> qml.math.squeeze(qml.set_shots(c, shots=1)())
  0
  ```
````

Reply:
Seems that four back ticks help resolve webui issue